internal/services/ingestor/ingestor.go

```diff
@@ -145,6 +146,42 @@ func (i *IngestorImpl) IngestEvent(ctx context.Context, tenantId, key string, da
 	return event, nil
 }
 
+func (i *IngestorImpl) BulkIngestEvent(ctx context.Context, eventOpts []*repository.CreateEventOpts) ([]*dbsqlc.Event, error) {
+	ctx, span := telemetry.NewSpan(ctx, "bulk-ingest-event")
+	defer span.End()
+
+	events, err := i.eventRepository.BulkCreateEvent(ctx, &repository.BulkCreateEventOpts{
+		Events: eventOpts,
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	return events, nil
+
+	// if err == metered.ErrResourceExhausted {
```

Comment:
rm?

Reply:
looks like we just dont have metering, etc enabled so i assume this is wip

Reply:
yes, hadn't figured out the metering piece yet